models/stored/mined_tx.go

```diff
@@ -0,0 +1,65 @@
+package stored
+
+import (
+	"bytes"
+
+	"github.com/Worldcoin/hubble-commander/models"
+)
+
+const (
+	mtxCommitmentLen = 34
+)
+
+// TODO: Rename this to something else, "CompletedTx"? It doesn't just store mined txs,
+//       it also stores failed txs.
```

Comment:
Good point. This actually might be a good enough reason to extract a third `FailedTx` struct. 
This way you would be able to use the following keys:
- `PendingTx`: txHash
- `MinedTx`: <batchID, commitmentID, indexInCommitment> 
- `FailedTx`: txHash

The use of such composite key for `MinedTx` might be a bigger change. If that's easier, maybe simply use txHash for now and leave this change for the next PR?

Also, as transactions would be stored in `MinedTx` even before the batch has actually been mined on the blockchain, maybe it would be better to call this structure `BatchedTx` or `CommittedTx`. What do you think?

Reply:
Both make sense to me, adding a `FailedTx` is easy to do in this PR so I can break it out, and seems like it will make changing the primary key in a future PR easier. And I like `BatchedTx`, I'll use that :+1: